common/errors.go

```diff
@@ -15,6 +15,14 @@ import (
 	"github.com/rs/zerolog"
 )
 
+var errorLabelMode = "verbose"
+
+func SetErrorLabelMode(mode string) {
+	if mode == "compact" || mode == "verbose" {
+		errorLabelMode = mode
+	}
+}
```

Comment:
Use "const" variable types instead of strings for 'compact' and 'verbose' modes.